### DISCUSSION TITLE
硕士论文模版封面不显示学校logo

### DISCUSSION THREAD

**Discussion by severous** [note286/xduts#23] [Q&A] [answered]
我在overleaf编译，发现无法显示学校logo
<img width="1396" alt="截屏2022-12-17 15 17 30" src="https://user-images.githubusercontent.com/37959509/208230616-26f7e82b-eb21-4eb7-99c8-22ef0c5d7cf2.png">

**Comment by note286** [comment, marked as answer]
研究生学位论文没有西安电子科技大学的校徽，这已经是第二个问的了，为啥会认为研究生学位论文里有校徽呢？

**Comment by severous** [comment]
知道了，谢谢，惯性思维了



| |
***@***.***
|
|
***@***.***
|




---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年12月17日 15:23 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [note286/xduts] 硕士论文模版封面不显示学校logo (Discussion #23) |

研究生学位论文没有西安电子科技大学的校徽，这已经是第二个问的了，为啥会认为研究生学位论文里有校徽呢？

**Reply by note286** [reply]
手册里已经提及了

> 请在阅读《西安电子科技大学研究生学位论文模板（2015 年修订版）-2019.03修订》后再使用`xdupgthesis`文档类。
